### ISSUE TITLE
Practicas 4 y 5

### ISSUE THREAD

**Issue by samtzai** [master-incar-ia/practicas-ia-2526-fernando-joseba#2]
Veo que las conclusiones que sacáis no se corresponden con lo que veo yo en las gráficas. Por ejemplo, vemos que el validation loss no baja a partir de 30, pero la grafica solo llega hasta 30.  y ponéis Habéis entendido por qué se usan logits y no se usa la activación en la capa final?. Corregid estos problemas e indicadme aquello que cambiéis. No se trata de poner copilot a todo, sino que saquéis conlclusiones de los resultados que obteneis.
